File: src/main/java/no/nav/veilarbvedtaksstotte/domain/oyeblikksbilde/OyeblikksbildeEgenvurderingDto.java
package no.nav.veilarbvedtaksstotte.domain.oyeblikksbilde;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;

@Data
@Accessors(chain = true)
@NoArgsConstructor
@AllArgsConstructor
public class OyeblikksbildeEgenvurderingDto {
    public EgenvurderingData data;
    public boolean journalfort;

    public OyeblikksbildeEgenvurderingDto setData(EgenvurderingData data) {
        this.data = data;
        return this;
    }

    public OyeblikksbildeEgenvurderingDto setJournalfort(boolean journalfort) {
        this.journalfort = journalfort;
        return this;
    }

    public EgenvurderingData getData() {
        return data;
    }

    public boolean isJournalfort() {
        return journalfort;
    }
}

File: src/main/java/no/nav/veilarbvedtaksstotte/domain/oyeblikksbilde/OyeblikksbildeEgenvurderingDto.kt
package no.nav.veilarbvedtaksstotte.domain.oyeblikksbilde

data class OyeblikksbildeEgenvurderingDto(
    val sistOppdatert: String? = null,
    val svar: List<Svar>? = null,
) {
    data class Svar(
        val spm: String,
        val svar: String? = null,
        val oppfolging: String? = null,
        val dialogId: String? = null,
    )
}
